src/components/shared/Project/FundingCycleDetailsCard.tsx

```diff
@@ -1,14 +1,54 @@
-import { ExclamationCircleOutlined } from '@ant-design/icons'
-import { Trans } from '@lingui/macro'
+import {
+  ExclamationCircleOutlined,
+  ClockCircleOutlined,
+} from '@ant-design/icons'
+import { t, Trans } from '@lingui/macro'
 import { Collapse, Tooltip } from 'antd'
 import CollapsePanel from 'antd/lib/collapse/CollapsePanel'
 import { ThemeContext } from 'contexts/themeContext'
 import { BigNumber } from '@ethersproject/bignumber'
 import { useContext } from 'react'
 import { detailedTimeUntil } from 'utils/formatTime'
+import { BallotState } from 'models/v2/fundingCycle'
+
+import { Badge, BadgeVariant } from '../Badge'
 
 const COLLAPSE_PANEL_KEY = 'funding-cycle-details'
 
+function BallotStateBadge({ ballotState }: { ballotState: BallotState }) {
+  // only show badge for ballot states 0 and 2 (don't show if ballot is 'approved'.)
+  const ballotStateVariantMap: { [k in BallotState]?: BadgeVariant } = {
+    0: 'warning',
+  }
+
+  const ballotStateLabelMap: { [k in BallotState]?: string } = {
+    0: 'Pending',
+  }
+
+  const ballotStateTooltips: { [k in BallotState]?: string } = {
+    0: t`This proposed funding cycle reconfiguration has an active ballot and isn't yet approved.`,
```

Comment:
The proposed reconfiguration shown below hasn't passed the x-day delay period yet. It may not take effect in the upcoming funding cycle.

Reply:
@johnnyd-eth 
<img width="435" alt="Screen Shot 2022-07-01 at 1 26 18 PM" src="https://user-images.githubusercontent.com/94939382/176820476-26c93077-d002-4499-a683-bd2af5db6b1f.png">

Reply:
SicK